Missionframework/modules/05_garrison/fnc/fn_garrison_reInitSectors.sqf

```diff
@@ -0,0 +1,78 @@
+/*
+    KPLIB_fnc_garrison_reInitSectors
+
+    File: fn_garrison_reInitSectors.sqf
+    Author: KP Liberation Dev Team - https://github.com/KillahPotatoes
+    Date: 2018-11-27
+    Last Update: 2018-11-27
+    License: GNU General Public License v3.0 - https://www.gnu.org/licenses/gpl-3.0.html
+
+    Description:
+        Reinitializes all affected sectors, when preset was changed during the campaign.
+
+    Parameter(s):
+        _preset - Variable name of the changed preset [STRING, defaults to ""]
+
+    Returns:
+        Function reached the end [BOOL]
+*/
+
+params [
+    ["_preset", "", [""]]
+];
+
+// Only execute on server, with valid parameter and not in mission init
+if (!(time > 0) || !isServer || (_preset isEqualTo "")) exitWith {};
```

Comment:
I know the handler is only added in a `isServer` category, I just thought it should be also a failsafe for manual local/global execution via debug console by server admins.